src/main/java/net/masterthought/cucumber/json/Result.java

```diff
@@ -28,4 +28,8 @@ public String getFormatedDuration() {
     public String getErrorMessage() {
         return errorMessage;
     }
+
+    public final String getErrorMessageTitle() {
+        return errorMessage.split("[\\r\\n]+")[0];
```

Comment:
If we put into pattern "\s" we got only first word from message. Current pattern returns first line. For example, we have exception. In case with "\s" pattern we got only type of exception, but with current pattern we got type with message in one line. I think it is more useful.